app/assets/javascripts/discourse/app/components/bookmark-list.hbs

```diff
@@ -7,7 +7,43 @@
       <thead class="topic-list-header">
         {{#if this.site.desktopView}}
           <PluginOutlet @name="bookmark-list-table-header">
-            <th class="topic-list-data">{{i18n "topic.title"}}</th>
+
+            {{#if this.bulkSelectEnabled}}
+              <th class="bulk-select topic-list-data">
+                {{raw
+                  "flat-button"
+                  class="bulk-select"
+                  icon="list"
+                  title="bookmarks.bulk.toggle"
+                }}
+              </th>
+            {{/if}}
+            <th class="topic-list-data">
+              {{#if this.bulkSelectEnabled}}
+                <span class="bulk-select-topics">
+                  {{~#if this.canDoBulkActions}}
+                    {{raw
```

Comment:
Can you try to use the Glimmer component directly here? For context, raw handlebars components are legacy elements, we are rtying to move away from them. (There's no way for you to have known this before...)

I suspect this may also help with the dialog test issue.